docs/examples/apt-cacher-ng.md

```diff
@@ -80,6 +83,15 @@ which obey `http_proxy`, such as `curl`, `wget` and others:
 might need to do it and you can do it from your `Dockerfile`
 too.
 
+**Option 4** is to link Debian-containers to proxy server using following command:
+
+    $ docker run -i -t --link test_apt_cacher_ng:apt_proxy -e http_proxy=http://apt_proxy:3142/ debian bash
+
+Here, the value provided to the `--link` flag is source_container_name:container_alias_name,
+where test_apt_cacher_ng is source_container_name and apt_proxy is container_alias_name.
+test_apt_cacher_ng was the name of first container that we launched earlier. The container_alias_name
+could be any name of your choice.
```

Comment:
I think we should try to avoid explaining how link-aliases work.

@moxiegirl may have some suggestions on how to re-word this